measurements/api/measurements.py

```diff
@@ -213,20 +206,21 @@ def list_measurements(
     # boolean arguments, that is logically the same of applying no filtering at
     # all.
     if failure is not None:
-        if set(clean_list_arg(failure)) == set(['true', 'false']):
+        if set(failure) == set(['true', 'false']):
```

Comment:
btw, that can be written as `{'true', 'false'}`

Reply:
Yes that is equivalent, however maybe it's clearer that we are comparing sets if the list is cast to a set explicitly.